source/_components/telegram_bot.markdown

```diff
@@ -79,3 +100,118 @@ action:
   - service: homeassistant.turn_off
     entity_id: switch.vision_zm1601eu5_battery_operated_siren_switch_9_0
 ```
+
+### {% linkable_title Sample automations with callback queries and inline keyboards %}
+
+Quick example to show some of the callback capabilities of inline keyboards with a dumb automation consisting in a simple repeater of normal text that presents an inline keyboard with 3 buttons: 'EDIT', 'NO' and 'REMOVE BUTTON':
+- Pressing 'EDIT' changes the sended message.
+- Pressing 'NO' only shows a brief notification (answering the callback query).
+- Pressing 'REMOVE BUTTON' changes the inline keyboard removing that button.
+
+Text repeater:
+```yaml
+- alias: 'telegram bot that repeats text'
+  hide_entity: true
+  trigger:
+    platform: event
+    event_type: telegram_text
+  action:
+    - service: notify.telegram
+      data_template:
+        title: '*Dumb automation*'
+        target: '{{ trigger.event.data.user_id }}'
+        message: 'You said: ``` {{ trigger.event.data.text }} ```'
+        data:
+          disable_notification: true
+          inline_keyboard:
+            - '/edit,/NO'
+            - '/remove button'
+```
+
+Message editor:
+```yaml
+- alias: 'telegram bot last sended msg edit'
```

Comment:
Preference of using capital letter of Telegram since it's used as a name throughout this documentation.